src/test/java/tests/FilterEvalTests.java

```diff
@@ -37,6 +44,17 @@ public boolean matches(String filterExpr, TurboIssue issue) {
         return Qualifier.process(empty, Parser.parse(filterExpr), issue);
     }
 
+    /**
+     * Helper method for testing SemanticException thrown by different qualifiers
+     */
+    public void verifySemanticException(QualifierType type, String invalidInput) {
+        TurboIssue issue = new TurboIssue(REPO, 1, "");
+        thrown.expect(SemanticException.class);
+        thrown.expectMessage(String.format(SemanticException.ERR_MSG, type, 
+                type.getValidInputs()));
```

Comment:
See line wrapping guidelines in this doc https://github.com/nus-cs2103/website/blob/master/contents/coding-standards-java.md